src/main/java/ru/yandex/qatools/fsm/impl/YatomataImpl.java

```diff
@@ -15,7 +15,7 @@
 /**
  * @author Ilya Sadykov
  */
-public class YatomataImpl<T> implements Yatomata<T> {
+class YatomataImpl<T> implements Yatomata<T> {
```

Comment:
What is the expected way of using Yatomata classes with Spring (i.e. when FSM class has autowired fields)? This change breaks my current implementation: https://github.com/meridor/perspective-backend/blob/master/perspective-worker/src/main/java/org/meridor/perspective/worker/processor/impl/FSMBuilderAwareImpl.java
The use case of the class above is:
https://github.com/meridor/perspective-backend/blob/master/perspective-worker/src/main/java/org/meridor/perspective/worker/processor/ImagesProcessor.java#L50